utils/createAccount.ts

```diff
@@ -0,0 +1,84 @@
+import { startRegistration } from "@simplewebauthn/browser";
+import { ApiKeyStamper } from "@turnkey/api-key-stamper";
+import { TurnkeyClient, createActivityPoller } from "@turnkey/http";
+import { deviceName } from "expo-device";
+import { UAParser } from "ua-parser-js";
+
+import base64URLEncode from "./base64URLEncode";
+import { rpId, turnkeyAPIPrivateKey, turnkeyAPIPublicKey, turnkeyOrganizationId } from "./constants";
+import generateRandomBuffer from "./generateRandomBuffer";
+import handleError from "./handleError";
+import { registrationOptions, verifyRegistration } from "./pomelo/client";
+import uppercase from "./uppercase";
+
+export default async function createAccount() {
+  const challengeID = base64URLEncode(generateRandomBuffer());
+  const { challenge } = await registrationOptions(challengeID);
+  const name = `exactly, ${new Date().toISOString()}`;
```

Comment:
Cool thanks for replying! But it make sense having the date in the name? The user will see this names, right? Maybe it could be improved so the user can differentiate between keys. Just a suggestion :)